include/aspect/stokes_matrix_free.h

```diff
@@ -123,6 +128,17 @@ namespace aspect
       Table<2, SymmetricTensor<2, dim, VectorizedArray<number>>>
       newton_factor_wrt_strain_rate_table;
 
+      /**
+       * Table which stores the product of the pressure perturbation
+       * and the normalized gravity.
```

Comment:
This is a tensor on each boundary face, right (even the ones without free surface)? Mention this in the comment here.